### DISCUSSION TITLE
pop data from the start of the dataset

### DISCUSSION THREAD

**Discussion by journeytosilius** [perspective-dev/perspective#1878] [Q&A]
Wanted to ask, is it possible somehow to set the LIMIT option at the table creation but configure it to pop data from the "start" of the buffer to create a moving window ? Also, can somebody point me to where can I configure the viewer to have more than one graph open at the same time ? 

Thanks

**Comment by texodus** [comment]
This is already [how the `limit` parameter works](https://perspective.finos.org/docs/md/table.html#index-and-limit).

`<perspective-viewer>` can only show one `Table()`/`View()`.  There is separate element, `<perspective-workspace>`, which allows a dynamic pool of `<perspective-viewer>` to access a set of pre-loaded `Table()`;   the [`movies` example](https://bl.ocks.org/texodus/6b4dcebf65db4ebe4fe53a6de5ea0b48) as well as many others (search for `perspective-workspace`) are a good starting place for this.
